src/utils.js

```diff
@@ -0,0 +1,66 @@
+const fs = require("fs");
+const csv = require("csv-parser");
+var path = require("path");
+
+async function parseCsvFile(filePath) {
+  return new Promise((resolve, reject) => {
+    const results = [];
+    fs.createReadStream(filePath, { encoding: "utf-8" })
+      .pipe(csv())
+      .on("data", (data) => results.push(data))
+      .on("end", () => resolve(results))
+      .on("error", (error) => reject(error));
+  });
+}
```

Comment:
I am unable to determine which part of this parser ignores the headers and which part of it "splits" each row on the comma.

Reply:
that is just processing a stream. All that is handled for us thanks to this neat library https://www.npmjs.com/package/csv-parser

I just accumulate the "data" already in json format, and resolve the promise at the end of the stream